HTML-CSS-JS/html-css.md

```diff
@@ -264,196 +264,272 @@ If you have 4 elements, then you will get two rows and three columns. in the fir
 
 ---
 
+## Week 2 (Questions with Medium priority)
 
-
-
-
----
-
----
-
----
-
-Week 2 (Questions with Medium priority)
 Number of questions: Medium = 12.
 Estimate: ~30m*12 = 6h / 5d = 1h 12m per day.
 
+---
 
+## Question 13
+### 1.13 Ways to improve website performance Medium
+Fix Broken URLs, Put CSS at the Top and JavaScript at the Bottom, Optimize images, Remove Unused scripts/plugins/css, minify scripts, use cache correctly, check the quality of the provider
```

Comment:
Putting CSS at the top won't help, if the size of CSS is huge. Better to follow critical rendering path concept. 
What do you mean "provider"?